eve_sqlalchemy/tests/patch.py

```diff
@@ -18,6 +18,7 @@ def test_patch_objectid(self):
     def test_patch_null_objectid(self):
         pass
 
+    @pytest.mark.xfail(True, run=False, reason='not applicable to SQLAlchemy')
     def test_patch_defaults(self):
```

Comment:
You can remove the implementation of the tests too, we will not need it anymore. See e.g. `test_patch_null_objectid`.